File: TheQueue.Server.Core/Models/QueueTicket.cs
namespace TheQueue.Server.Core.Models
{
    public class QueueTicket
    {
        public string Name { get; set; }
        public int Ticket { get; set; }

        public QueueTicket(string name, int ticketNumber)
        {
            Name = name;
            Ticket = ticketNumber;
        }
    }
}

File: TheQueue.Client.Common/src/main/java/Models/BroadcastMessages/QueueTicket.java
package Models.BroadcastMessages;

import com.fasterxml.jackson.annotation.JsonProperty;

public class QueueTicket {
    @JsonProperty
    private String Name;
    @JsonProperty
    private int Ticket;

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public int getTicket() {
        return Ticket;
    }

    public void setTicket(int ticket) {
        Ticket = ticket;
    }
}
